apps/meteor/app/file-upload/server/config/AmazonS3.ts

```diff
@@ -11,7 +11,17 @@ import '../../ufs/AmazonS3/server';
 
 const get: FileUploadClass['get'] = async function (this: FileUploadClass, file, req, res) {
 	const { query } = URL.parse(req.url || '', true);
-	const forceDownload = typeof query.download !== 'undefined';
+	let forceDownload = typeof query.download !== 'undefined';
+	if (!forceDownload) {
+		switch (query.contentDisposition) {
```

Comment:
When query.contentDisposition is none (inline, attachment) forceDownload will be false. If that's fine, ignore this comment, otherwise add a default case.

Reply:
Its fine because its the way it works today.